### PR TITLE
Testing: Jest and PhpUnit

### PR DESCRIPTION
**Pull request by joekarasek**
WooHoo!
Testing in both Jest and PhpUnit

`npm test`
`npm test:js`
`npm test:php`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,8 +38,11 @@
     "clean:lerna": "lerna clean -Y",
     "clean": "npm-run-all --parallel clean:*",
     "prepublishOnly": "npm run build",
+    "test:js": "jest --testPathIgnorePatterns 'sandbox' --testPathIgnorePatterns 'brightcove-player.test.js'",
     "snyk-protect": "snyk protect",
-
+    "test": "npm run test:js && npm run test:php",
```
Comment (EvanLovely) [inline package.json]: Change `test` to be `npm-run-all --serial test:*`

File: package.json [hunk 2]
```diff
@@ -38,8 +38,11 @@
...
+    "test:js": "jest --testPathIgnorePatterns 'sandbox' --testPathIgnorePatterns 'brightcove-player.test.js'",
+    "test:php": "cd packages/core-php && ./vendor/bin/phpunit --bootstrap vendor/autoload.php tests",
```
Comment (EvanLovely) [inline package.json]: Create a `scripts` object in `composer.json` in `core-php` that has `"test": "phpunit tests"` in it – it works *very* similar to `npm run` so you can `composer run test` for this.

**Comment by EvanLovely** [comment]
Looking good!! Got a few comment in there.

File: package.json (hunk 1 above)
Comment (joekarasek) [reply package.json]: 👍

File: package.json (hunk 2 above)
Comment (joekarasek) [reply package.json]: 👍

**Comment by sghoweri** [comment]
SO FREAKIN AWESOME! Super excited on what this work will help unlock for us down the road! 🥇

**Comment by EvanLovely** [comment]
@joekarasek Looks like the build is failing because phpunit 7 requires php 7.1 and our minimum requirement is php 7.0 - could you please downgrade phpunit?

**Comment by sghoweri** [comment]
@EvanLovely why not just update the minimum PHP version to 7.1?

**Comment by sghoweri** [comment]
^ just a matter of time that we'll need to do it anyways...

**Comment by EvanLovely** [comment]
I’m down with that. In fact the current versions of Drupal actually doesn’t recommend php 7.0:
https://www.drupal.org/docs/8/system-requirements/drupal-8-php-requirements

**Pull request merged**